resources/public/api/conf/2.0/application.yaml

```diff
@@ -0,0 +1,10973 @@
+                  creationReason: New
+                  accountId: '1234567890'
+                  firstSubscriptionDate: 2017-04-06
+              example-2:
+                value:
+                  investorId: '9876543210'
+                  creationReason: Transferred
+                  accountId: '1234567891'
+                  firstSubscriptionDate: 2017-04-06
+                  transferAccount:
+                    transferredFromAccountId: '8765432100'
+                    transferredFromLMRN: Z654321
+                    transferInDate: 2017-04-06
+        required: true
+      responses:
+        '201':
+          description: Created
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/lifetimeIsaManagerAccountsResponse'
+              examples:
+                example-1:
+                  value:
+                    status: 201
+                    success: true
+                    data:
+                      message: Account created
+                      accountId: '1234567890'
+                example-2:
+                  value:
+                    status: 201
+                    success: true
+                    data:
+                      message: Account transferred
+                      accountId: '1234567891'
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/lifetimeIsaErrorResponse'
+              examples:
+                badRequest:
+                  $ref: '#/components/examples/invalidLmrn'
+                invalidLmrn:
+                  $ref: '#/components/examples/invalidLmrn'
+        '401':
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/lifetimeIsaErrorResponse'
+              examples:
+                notOpenStatus:
+                  $ref: '#/components/examples/notOpenStatusUnauthorized'
+        '403':
+          description: Forbidden
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/lifetimeIsaErrorResponse'
+              examples:
+                investorNotFound:
+                  description: The investor details given do not match with HMRC’s records
+                  value:
+                    code: INVESTOR_NOT_FOUND
+                    message: 'invalid status: must create with open status'
+                outsideAgeRange:
+                  description: The investor is not eligible for a LISA account
+                  value:
+                    code: INVESTOR_ELIGIBILITY_CHECK_FAILED
+                    message: Can’t open a LISA if investor is older than 40 or younger than 18
+                complianceFail:
```

Comment:
Missing a description here, please add.

```
complianceFail:
  description: You cannot create or transfer a LISA account because the investor has failed a compliance check
  value:
    code: INVESTOR_COMPLIANCE_CHECK_FAILED
    message: You cannot create or transfer a LISA account because the investor has failed a compliance check
```